mathesar/api/serializers/databases.py

```diff
@@ -7,12 +7,17 @@
 
 
 class DatabaseSerializer(MathesarErrorMessageMixin, serializers.ModelSerializer):
+    name = serializers.CharField()
+    db_username = serializers.CharField()
```

Comment:
Is there a reason to explicitly specify the field types here, Doesn't drf infer from the model? It would be good to add comments on the need for overriding the field type